kotlin-sqlobject/src/main/kotlin/org/jdbi/v3/sqlobject/kotlin/RegisterKotlinMapper.kt

```diff
@@ -14,20 +14,15 @@
 package org.jdbi.v3.sqlobject.kotlin
 
 import org.jdbi.v3.sqlobject.config.ConfiguringAnnotation
-import java.lang.annotation.ElementType
-import java.lang.annotation.Retention
-import java.lang.annotation.RetentionPolicy
-import java.lang.annotation.Target
 import kotlin.reflect.KClass
 
 /**
  * Registers a KotlinMapper for a specific kotlin class
  */
-@Retention(RetentionPolicy.RUNTIME)
+@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
 @ConfiguringAnnotation(RegisterKotlinMapperImpl::class)
-@Target(ElementType.TYPE, ElementType.METHOD)
+@kotlin.annotation.Target(AnnotationTarget.TYPE, AnnotationTarget.CLASS)
```

Comment:
I don't know enough about kotlin to be really helpful but should't that be `AnnotationTarget.FUNCTION`?

Reply:
You're completely right. Done